lib/modules/manager/glasskube/schema.ts

```diff
@@ -0,0 +1,31 @@
+import { z } from 'zod';
+
+export const Package = z.object({
+  apiVersion: z.string().startsWith('packages.glasskube.dev/'),
+  kind: z.literal('Package').or(z.literal('ClusterPackage')),
+  spec: z.object({
+    packageInfo: z.object({
+      name: z.string(),
+      version: z.string(),
+      repositoryName: z.string().optional(),
+    }),
+  }),
+});
+
+export const PackageRepository = z.object({
+  apiVersion: z.string().startsWith('packages.glasskube.dev/'),
+  kind: z.literal('PackageRepository'),
+  metadata: z.object({
+    name: z.string(),
+    annotations: z.object({}).catchall(z.string()).optional(),
```

Comment:
```suggestion
    annotations: z.record(z.string(), z.string()).optional(),
```
Based on your usage in the code a Record should be a better fit

Reply:
Ahh, yes that is much better! I tried `map(string(), string())` but that didn't work for some reason…